it/repositories/ListRepositorySpec.scala

```diff
@@ -66,42 +60,91 @@ class ListRepositorySpec
         .many(data)
     }.futureValue
 
-  "getList" - {
+  "getListByName" - {
 
-    "must handle" - {
+    "must return list of JsObjects when" - {
 
       "single record found" in {
 
-        seedData(database, Seq(sampleDataSet1))
+        forAll(arbitrary[GenericListItem], arbitrary[ListName]) {
+          (genericListItem, listName) =>
 
-        val repository = app.injector.instanceOf[ListRepository]
-        val result     = repository.getList(ListName("AdditionalInformationIdCommon"), MetaData("", LocalDate.now))
-        result.futureValue mustBe List(sampleDataSet1)
+            val setListName = genericListItem.copy(listName = listName)
+            val itemToJson = Json.toJsObject(setListName) ++ Json.obj("_id" -> BSONObjectID.generate.toString)
+
+            seedData(database, Seq(itemToJson))
+
+            val repository = app.injector.instanceOf[ListRepository]
+            val result = repository.getListByName(listName, MetaData("", LocalDate.now))
+
+            result.futureValue mustBe List(itemToJson)
+
+            database.flatMap(_.drop()).futureValue
```

Comment:
I don't think we need these drops, especially since they will blow away the indexing setup for the collection between tests